### PR TITLE
Bump rubygem-unf_ext to 0.0.7.6-1

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by evgeni** [comment]
well, it builds now :woman_shrugging:

File: packages/foreman/rubygem-unf_ext/rubygem-unf_ext.spec [hunk 1]
```diff
@@ -1,90 +1,113 @@
...
 
 %install
 mkdir -p %{buildroot}%{gem_dir}
-cp -a .%{gem_dir}/* %{buildroot}/%{gem_dir}
-
-mkdir -p %{buildroot}%{gem_extdir_mri}
 cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
+mkdir -p %{buildroot}%{gem_extdir_mri}/
```
Comment (ehelms) [inline packages/foreman/rubygem-unf_ext/rubygem-unf_ext.spec:68]: Can the line after get updated to match our scl gem2rpm template?

File: packages/foreman/rubygem-unf_ext/rubygem-unf_ext.spec [hunk 2]
```diff
@@ -1,90 +1,113 @@
...
+
 %files
 %dir %{gem_instdir}
-%{gem_libdir}
-%doc %{gem_instdir}/LICENSE.txt
 %{gem_extdir_mri}
-%exclude %{gem_instdir}/ext
+%exclude %{gem_instdir}/.gitignore
+%exclude %{gem_instdir}/.travis.yml
+%exclude %{gem_instdir}/.document
```
Comment (ehelms) [inline packages/foreman/rubygem-unf_ext/rubygem-unf_ext.spec:92]: I don't expect other gems to have this file, but adding to the ignore list int he template will prevent future occurrances.

File: packages/foreman/rubygem-unf_ext/rubygem-unf_ext.spec [hunk 3]
```diff
@@ -1,90 +1,113 @@
...
+# Ideally, this would be something like this:
+# GEM_PATH="%{buildroot}%{gem_dir}:$GEM_PATH" ruby -e "require '%{gem_name}'"
+# But that fails to find native extensions on EL8, so we fake the structure that ruby expects
+mkdir gem_ext_test
+cp -a %{buildroot}%{gem_dir} gem_ext_test/
+mkdir -p gem_ext_test/gems/extensions/%{_arch}-%{_target_os}/$(ruby -r rbconfig -e 'print RbConfig::CONFIG["ruby_version"]')/
+cp -a %{buildroot}%{gem_extdir_mri} gem_ext_test/gems/extensions/%{_arch}-%{_target_os}/$(ruby -r rbconfig -e 'print RbConfig::CONFIG["ruby_version"]')/
+GEM_PATH="./gem_ext_test/gems:$GEM_PATH" ruby -e "require '%{gem_require_name}'"
+rm -rf gem_ext_test
+%{?scl:EOF}
```
Comment (ehelms) [inline packages/foreman/rubygem-unf_ext/rubygem-unf_ext.spec:85]: Are we ready to add this to our template?

Comment (evgeni) [reply packages/foreman/rubygem-unf_ext/rubygem-unf_ext.spec:85]: If you'd have been to scrum... ;) (But I also mentioned that in one of the other PRs)
The three PRs in review are there to see if y'all like it, and then add it to the template and redo all the other PRs.

File: packages/foreman/rubygem-unf_ext/rubygem-unf_ext.spec (hunk 1 above)
Comment (evgeni) [reply packages/foreman/rubygem-unf_ext/rubygem-unf_ext.spec]: not sure what you mean, the line after is the one placing the so in the wrong folder, which I explicitly reverted :)

Comment (evgeni) [reply packages/foreman/rubygem-unf_ext/rubygem-unf_ext.spec]: see https://github.com/theforeman/foreman-packaging/pull/5052

**Comment by ekohl** [comment]
As the author of the PR GH won't allow me to approve, but :+1:

**Comment by ekohl** [comment]
Looks like this breaks repoclosure on plugins since there something requires `= 0.0.7.2`.

**Comment by evgeni** [comment]
```console
% koji untag-build foreman-nightly-rhel7 tfm-rubygem-unf_ext-0.0.7.6-1.el7
% koji untag-build foreman-nightly-el8 rubygem-unf_ext-0.0.7.6-1.el8
```

the rest is in https://github.com/theforeman/foreman-packaging/pull/5061

**Pull request merged**